skins/laika/src/components/shared/AmountSelection.vue

```diff
@@ -52,6 +53,7 @@ export default Vue.extend( {
 			type: String,
 			default: '',
 		},
+		caption: String,
```

Comment:
This might create an error if you leave caption undefined in the parent component. Maybe add a default value.